<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>crearMesas</class>
 <widget class="QDialog" name="crearMesas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(237, 232, 208);
</string>
  </property>
  <widget class="QPushButton" name="desactivarMesa">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(148, 17, 0, 234);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Desactivar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Georgia&quot;;
color: rgb(45, 81, 40);
</string>
   </property>
   <property name="text">
    <string>Numero de Mesa: </string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 24pt &quot;Georgia&quot;;
color: rgb(45, 81, 40);
</string>
   </property>
   <property name="text">
    <string>MANEJAR MESAS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="activarMesa">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 81, 40);
color: white;</string>
   </property>
   <property name="text">
    <string>Activar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
